exporter/tanzuobservabilityexporter/README.md

```diff
@@ -21,7 +21,7 @@ This exporter supports sending traces to [Tanzu Observability](https://tanzu.vmw
 - Application identity tags, which are [required by Tanzu Observability](https://docs.wavefront.com/trace_data_details.html#how-wavefront-uses-application-tags), are added if they are missing.
   - `application` is set to "defaultApp".
   - `service` is set to "defaultService".
-- A `source` field is required in a [Tanzu Observability Span](https://docs.wavefront.com/trace_data_details.html#span-fields). The `source` is set to the first matching OpenTelemetry Resource Attribute from the list below. The matched Attribute is excluded from the resulting Tanzu Observability Span tags to reduce duplicate data. If none of the Attributes exist on the Resource, the hostname of the OpenTelemetry Collector is used as a default for `source`.
+- A `source` field is required in a [Tanzu Observability Span](https://docs.wavefront.com/trace_data_details.html#span-fields) and [Tanzu Observability Metrics](https://docs.wavefront.com/trace_data_details.html#red-metrics-for-tracing-traffic). The `source` is set to the first matching OpenTelemetry Resource Attribute from the list below. The matched Attribute is excluded from the resulting Tanzu Observability Span/Metrics tags to reduce duplicate data. If none of the Attributes exist on the Resource, the hostname of the OpenTelemetry Collector is used as a default for `source`.
```

Comment:
I think the link for "Tanzu Observability Metrics" should point to https://docs.wavefront.com/wavefront_data_format.html#wavefront-data-format-fields.

Reply:
Resolved.